lib/pbench/test/unit/server/conftest.py

```diff
@@ -230,7 +233,7 @@ def fake_email_validator(monkeypatch):
     Set up a mock for the email validator so we control failure modes.
     """
 
-    def fake_email(value: str, **kwargs) -> ValidatedEmail:
+    def fake_email(value: str, **_kwargs) -> ValidatedEmail:
```

Comment:
Somehow, `**_kwargs` just looks so weird my eye keeps going back to it. Are you suggesting that a linter is now complaining about non-undered unused formal parameters, or was this just a "what the heck" moment?

Reply:
The linter complains that `kwargs` is an unused parameter (which, it is, more or less, but that doesn't mean we can omit it...).  The underscore prefix follows a convention which confirms to the linter that the parameter is intentionally unused, and so it stops complaining.

We _could_ name it something other than `_kwargs`...provided that it starts with an underscore...got a preference?  (We could use just a bare underscore, but I think that `_kwargs` is better.)

Reply:
I didn't actually ever notice that the linter was inclined to treat `_` prefixes to special rules. Huh. So as ugly as it looks, well, fine. (Ish.)